### PR TITLE
Add strict mode of class example

### PR DESCRIPTION
**Pull request by jzz567**
### Description
Add an example for strict mode strict mode.

### Motivation
 Class's strict mode example is missing. Hope this example can help people understand more about class's strict mode.

### Additional details

This example compare the object and class in strict mode, because in non strict mode, `this` will point to `window` object,
and in strict mode, `this` will point to `undefined`

### CODE REVIEW

File: files/en-us/web/javascript/reference/strict_mode/index.md [hunk 1]
```diff
@@ -73,6 +73,36 @@ export default myStrictFunction;
...
+
+
+class TestClassStrictMode {
+    printThis() {
+        console.log(this);
+    }
+}
+const testClassStrictMode = new TestClassStrictMode();
+printThis = testClassStrictMode.printThis;
+printThis(); // undefined, because printThis() is in a class, strict mode enabled by default.
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/strict_mode/index.md:103]: This seems too long to just illustrate classes are strict.

```suggestion
class C {
  // All code here is evaluated in strict mode
}

const C = class {
  // All code here is evaluated in strict mode
}

// Code here may not be in strict mode
```

Comment (jzz567) [reply files/en-us/web/javascript/reference/strict_mode/index.md]: Thanks @Josh-Cena , I have remove 1 code block from this example, could you please take a look again? Thanks!

Comment (Josh-Cena) [reply files/en-us/web/javascript/reference/strict_mode/index.md]: This part does not illustrate in what particular way does a strict mode function behave. It simply tells you that this particular piece of source code is in strict mode. It's the exact same idea as the module example above. Again—I want to keep it short and to the point.

Comment (jzz567) [reply files/en-us/web/javascript/reference/strict_mode/index.md]: OK, maybe this example is better in a JS book, thanks!
I will update the code.

Comment (jzz567) [reply files/en-us/web/javascript/reference/strict_mode/index.md]: Hi, @Josh-Cena 
I have updated the example, could you please take a look again? 
Thanks!

**Review by Josh-Cena — APPROVED** [review]
This looks better. Thanks! Congratulations on landing your first PR here—welcome aboard!

**Pull request merged**